### ISSUE TITLE
missing files in 375.10 nvidia-settings sources

### ISSUE THREAD

**Issue by sl1pkn07** [NVIDIA/nvidia-settings#4]
```
   CC           libXNVCtrlAttributes/NvCtrlAttributesNvml.c
libXNVCtrlAttributes/NvCtrlAttributesNvml.c:36:18: fatal error: nvml.h: No such file or directory
 #include "nvml.h"
                  ^
compilation terminated.
```

**Comment by scaronni** [comment]
`nvml.h` was previously part of the GDK, and is now part of CUDA 8
But even compiling with the CUDA 8 header installed, compilation aborts as follows:

```
libXNVCtrlAttributes/NvCtrlAttributesNvml.c: In function 'NvCtrlNvmlGetGPUAttribute':
libXNVCtrlAttributes/NvCtrlAttributesNvml.c:795:21: error: unknown type name 'nvmlGpuVirtualizationMode_t'
                     nvmlGpuVirtualizationMode_t mode;
                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by sl1pkn07** [comment]
yep. after

```
sed 's|/usr/include/nvidia/gdk|/opt/cuda/include/|g' -i src/Makefile
```

show the @scaronni error

**Comment by sl1pkn07** [comment]
any update?

i think this set of changes is the culprit

https://github.com/NVIDIA/nvidia-settings/commit/7155801a4e8e162981fdb963cd5b76982a2deb76#diff-f57f2991a6aa25fe45d8036c51bf8b4d

greetings

**Comment by aaronp24** [comment]
A fix is in progress. It's taking a little while because nvml.h is build using a fairly complicated generator so we need to make sure the right version of it gets bundled with nvidia-settings.

**Comment by aaronp24** [comment]
Fixed with commit 168e17f53098254b4a5ab93eeb2f23c80ca1d97f.
